Jenkinsfile

```diff
@@ -56,33 +66,92 @@ def get_daos_packages(String distro) {
     return pkgs + "-" + daos_packages_version(distro)
 }
 
-def component_repos() {
+def pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos')
 }
 
-String el7_component_repos() {
+String el7_pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos-el7')
 }
 
-String leap15_component_repos() {
+String leap15_pr_repos() {
     return cachedCommitPragma(pragma: 'PR-repos-leap15')
 }
 
-def daos_repo() {
+String unit_repos() {
+    Map stage_info = parseStageInfo()
+    return unit_repos(stage_info['target'])
+}
+
+String unit_repos(String distro) {
```

Comment:
Isn't this really just `pr_repos()` since it's just aggregating the global `PR-repos` and the distro specific pr-repos?

It seems like `pr_repos()` should do that aggregation itself and then instead of using `unit_repos()` elsewhere in this file, you would just use `pr_repos()`.  I can't think of any case where we'd want the separate values all-distro `PR-repos` and distro-specific pr-repos.